src/main/java/net/sf/jabref/logic/importer/fetcher/IsbnFetcher.java

```diff
@@ -43,6 +49,9 @@
 
     private static final String URL_PATTERN = "http://www.ebook.de/de/tools/isbn2bibtex?";
 
+    private final ProtectTermsFormatter protectTermsFormatter = new ProtectTermsFormatter(new ProtectedTermsLoader(new ProtectedTermsPreferences(ProtectedTermsLoader.getInternalLists(), Collections.emptyList(), Collections.emptyList(), Collections.emptyList())));
```

Comment:
Ok, in this case just remove them. They can be latter added as configurable cleanup operations if this is needed.